src/constants/initialHomebaseTabsConfig.tsx

```diff
@@ -0,0 +1,1139 @@
+      fidgetShadow: "none",
+      font: "Exo",
+      fontColor: "#000000",
+      headingsFont: "Work Sans",
+      headingsFontColor: "#000000",
+      musicURL: "https://www.youtube.com/watch?v=dMXlZ4y7OK4&t=1804",
+    },
+  },
+  fidgetInstanceDatums: {
+    "iframe:45e7e2e0-2b6a-467a-814e-95995089a913": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          background: "var(--user-theme-fidget-background)",
+          fidgetBorderColor: "var(--user-theme-fidget-border-color)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
+          fidgetShadow: "var(--user-theme-fidget-shadow)",
+          size: 0.9,
+          url: "https://space.nounspace.com",
+        },
+      },
+      fidgetType: "iframe",
+      id: "iframe:45e7e2e0-2b6a-467a-814e-95995089a913",
+    },
+    "feed:a3be1c38-b83a-42f7-abce-014ae049ffc7": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          Xhandle: "thenounspace",
+          background: "var(--user-theme-fidget-background)",
+          channel: "",
+          feedType: "following",
+          fidgetBorderColor: "var(--user-theme-fidget-border-color)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
+          fidgetShadow: "var(--user-theme-fidget-shadow)",
+          filterType: "fids",
+          fontColor: "var(--user-theme-font-color)",
+          fontFamily: "var(--user-theme-font)",
+          selectPlatform: {
+            icon: "/images/twitter.avif",
+            name: "The other app",
+          },
+          style: "light",
+          urlColor: "blue",
+          users: "",
+        },
+      },
+      fidgetType: "feed",
+      id: "feed:a3be1c38-b83a-42f7-abce-014ae049ffc7",
+    },
+    "iframe:0f8fd96c-8cb7-4aac-86de-a2d184a96302": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          background: "var(--user-theme-fidget-background)",
+          fidgetBorderColor: "var(--user-theme-fidget-border-color)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
+          fidgetShadow: "var(--user-theme-fidget-shadow)",
+          size: 0.6,
+          url: "https://nounswap.wtf",
+        },
+      },
+      fidgetType: "iframe",
+      id: "iframe:0f8fd96c-8cb7-4aac-86de-a2d184a96302",
+    },
+    "iframe:d65e4ea9-d6ac-48d0-8eaf-d4c2c4187f7a": {
+      config: {
+        data: {},
+        editable: true,
+        settings: {
+          background: "var(--user-theme-fidget-background)",
+          fidgetBorderColor: "var(--user-theme-fidget-border-color)",
+          fidgetBorderWidth: "var(--user-theme-fidget-border-width)",
+          fidgetShadow: "var(--user-theme-fidget-shadow)",
+          url: "https://irb0gie.vercel.app",
+        },
+      },
+      fidgetType: "iframe",
+      id: "iframe:d65e4ea9-d6ac-48d0-8eaf-d4c2c4187f7a",
+    },
+  },
+  isEditable: true,
+  fidgetTrayContents: [],
+};
```

Comment:
These are contents of a draft/test tab, we can remove

Reply:
If I remove I get a type error 

```bash 
[{
	"resource": "/Users/web3warrior/Desktop/code/nounspace.ts/src/constants/initialHomebaseTabsConfig.tsx",
	"owner": "typescript",
	"code": "2739",
	"severity": 8,
	"message": "Type '{ layoutID: string; layoutDetails: { layoutConfig: { layout: { h: number; i: string; maxH: number; maxW: number; minH: number; minW: number; moved: boolean; resizeHandles: string[]; static: boolean; w: number; x: number; y: number; }[]; }; layoutFidget: string; }; theme: { ...; }; fidgetInstanceDatums: { ...; }; }' is missing the following properties from type 'SpaceConfig': isEditable, fidgetTrayContents",
	"source": "ts",
	"startLineNumber": 7,
	"startColumn": 14,
	"endLineNumber": 7,
	"endColumn": 34
}]

```